contrib/largeNbDicts/largeNbDicts.c

```diff
@@ -559,7 +559,7 @@ static int benchMem(slice_collection_t dstBlocks,
         CONTROL(BMK_isSuccessful_runOutcome(outcome));
 
         BMK_runTime_t const result = BMK_extract_runTime(outcome);
-        U64 const dTime_ns = result.nanoSecPerRun;
+        U64 const dTime_ns = (U64)result.nanoSecPerRun;
```

Comment:
I believe it would actually be better to make `dTime_ns` a `double`,
this would avoid a useless double-conversion.

I think this issue happened because `nanoSecPerRun` changed from a `U64` to a `double` type 
well after creation of `largeNbDicts.c` test program.

Reply:
thanks, let me update it